auth/src/main/java/com/firebase/ui/auth/data/remote/SignInKickstarter.java

```diff
@@ -182,7 +184,13 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
                 } else if (response.isSuccessful()) {
                     setResult(Resource.forSuccess(response));
                 } else {
-                    setResult(Resource.<IdpResponse>forFailure(response.getError()));
+                    if (response.getError().getErrorCode() ==
```

Comment:
nit: rather than:

```
else {
   if {
   } else {
   }
} 
```

Should just be another `else if` followed by a final `else`

Reply:
Done.